scepter.ts

```diff
@@ -96,6 +96,11 @@ const loadModule = (name: string) => {
         })
       })
     }
+    if (module.events) {
+      module.events.map((event: Event) => {
+        client.on(event.trigger, event.event)
```

Comment:
Does the `event` callback receive any arguments? re: Event type changes

Reply:
(discussed in chat)